config/src/comments.rs

```diff
@@ -228,7 +228,7 @@ fn comments() -> HashMap<String, String> {
 #ban_window = 10800
 
 #maximum number of peers
-#peer_max_count = 25
+#peer_max_count = 125
```

Comment:
Shouldn't this option be uncommented? Otherwise, it's not really a default but more like a suggestion.

Reply:
Or is the default driven by PEER_MAX_COUNT below?

Reply:
The default is driven by `PEER_MAX_COUNT`. Ideally config comments that are generated should be driven by that, as @antiochp mentioned, but it's a little out of scope for this PR.

Reply:
Opened https://github.com/mimblewimble/grin/issues/2781